src/selectors/layers.js

```diff
@@ -41,7 +39,7 @@ export const getLayers = createSelector(
       const nextID = layerIDs[i + 1];
       return {
         id,
-        name: layerName[id],
+        name: id,
```

Comment:
As mentioned in another comment, I think that you should either keep `name: layerName[id],`, or delete the `id` field and just use `name` for everything.